models/user.go

```diff
@@ -53,6 +55,18 @@ func GetUserByUsername(username string) (User, error) {
 	return u, err
 }
 
+// UpdateLastLogin updates the timestamp for the last successful login for a user
```

Comment:
What do you think about removing this method and instead just update `LastLogin` then call `models.PutUser(&u)` back in the `Login` handler?

Reply:
Ah, much more elegant approach, good call @jordan-wright! I've made that change and committed.